geomstats/rotations.py

```diff
@@ -280,9 +280,9 @@ def riemannian_log(rot_vec,
 
     This gives a tangent vector at point ref_point.
 
-    :params rot_vec: 3D rotation vector
+    :param rot_vec: 3D rotation vector
     :param ref_point: 3D rotation vector
-    :returns a: 3D rotation vector, tangent vector at ref_point
+    :returns rot_vec_log: 3D rotation vector, tangent vector at ref_point
```

Comment:
What's the standard for return syntax for this? It doens't make sense to pass a param to the return argument right?

Reply:
You mean the stantard syntax in docstrings for the return argument? I'm not sure, but I could use Sphinx' syntax (close to what I have now) to then automatically generate documentation?
https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html
http://www.sphinx-doc.org/en/stable/index.html
Unless you've heard about another documentation generator?

Reply:
I was assuming you were using the sphinx syntax already. There shouldn't be any return parameter.

Reply:
Really? In the first link above they put some.